test/composables/useTheme.test.ts

```
@@ -114,6 +120,10 @@
         defaultBaseUrl: 'https://app.local',
         getServers: ({ method, path, operation }) => (`https://app.local/${method}${path}`),
       },
+      server: {
+        allowCustomServers: true,
+        getServers: ({ method, path, operation }) => [`https://app.local/${method}${path}`],
```

ESLint

[error] 'operation' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)